### ISSUE TITLE
NavList clean up

### ISSUE THREAD

**Issue by ChuckBTaylor** [mwkwsd/sensensomething#100]
Just looking through it, have some ideas for clean up:

* Make roles and genres one list (instead of two lists combined later).  Current Figma file has all categories as one group.
```ts
  const subheaderLinkItems: SubheaderLinksProps[] = [
    { items: roles, navigate },
    { items: genres, navigate },
  ]
```

* Refactor to get rid of `SubheaderLinks` component.  I think it be `NavLinks` and then another `NavLinks`.  The styling for them is the same.  Also the verbiage for `SubheaderLinks` is confusing now since we don't have "Subheaders" any more.

**Comment by mwilkins74** [comment]
100%. 
The Subheader part was created when ROLE and GENRE subheaders were present. 

Definite refactor and clean up here.

**Comment by ChuckBTaylor** [comment]
Oh ya. No questions as to why it is that way.

**Comment by mwilkins74** [comment]
Closed in #102
